test/functional/core/startup_spec.lua

```diff
@@ -219,6 +219,29 @@ describe('startup', function()
     clear{args={'--embed'}}
     eq(2, eval('1+1'))
   end)
+
+  it('message display ends with enter #7967', function()
```

Comment:
#7967 is fixed on master, right? So we can add this test already.

Reply:
Yes, #7967 has been fixed by # 9884.